src/styles/blocks/gallery.scss

```diff
@@ -0,0 +1,154 @@
+.gallery {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  gap: 40px;
+
+  &__title {
+    font-size: 48px;
+    line-height: 48px;
+    font-weight: 700;
+    letter-spacing: -1.5px;
+    margin: 0  auto 58px;
+    position: relative;
+    color: #000;
+
+    &:after {
+      content: "";
+      width: 24px;
+      height: 23px;
+      position: absolute;
+      top: 100%;
+      left: 55px;
+      background-image: url("../images/crown.svg");
+      background-size: contain;
+      background-repeat: no-repeat;
+    }
+  }
+
+  &__item {
+    display: flex;
+    justify-content: space-between;
+    flex-direction: column;
+
+    &-photos {
+      width: 100%;
+      height: 256px;
+      display: flex;
+      justify-content: space-between;
+      gap: 20px;
+      margin-bottom: 24px;
+
+      @media (max-width: 360px) {
+        height: 131px;
+      }
+
+      &-photo_1 {
+        width: 66.6%;
+        background-size: contain;
+        background-repeat: no-repeat;
+        object-fit: cover;
+
+        @media (max-width: 1000px) {
+          width: 50%;
+        }
+      }
+
+      &-photo_2 {
+        width: calc(33.3% - 20px);
+        background-size: contain;
+        background-repeat: no-repeat;
+        object-fit: cover;
+
+        @media (max-width: 1000px) {
+          width: calc(50% - 20px);
+        }
+      }
+
+      &-photo_3 {
+        width: 50%;
+        background-size: contain;
+        background-repeat: no-repeat;
+        object-fit: cover;
+
+        @media (max-width: 1000px) {
+          width: 50%;
+        }
+      }
+
+      &-photo_4 {
+        width: calc(50% - 20px);
+        background-size: contain;
+        background-repeat: no-repeat;
+        object-fit: cover;
+
+        @media (max-width: 1000px) {
+          width: calc(50% - 20px);
+        }
+      }
+
+      &-photo_5 {
+        width: 33.3%;
+        background-size: contain;
+        background-repeat: no-repeat;
+        object-fit: cover;
+
+        @media (max-width: 1000px) {
+          width: 50%;
+        }
+      }
+
+      &-photo_6 {
+        width: calc(66.6% - 20px);
```

Comment:
Instead of creating 6 classes for photos with similar styles where you are repeating yourself, it's better to create 3 blocks of photos and calculate the width of the photos with :first/last-child (like you do: 
```
width: calc(66.6% - 20px);
width: calc(33.3% - 20px);
```
And better to make a variable for 20px and call it like `gap` or smth